### ISSUE TITLE
Doc: problems with straight double quotes in l3news

### ISSUE THREAD

**Issue by jfbu** [latex3/latex3#683]
Look at the quotes in this screenshot from ltnews (issue 9, March 2014):

![Capture d’écran 2020-02-26 à 15 07 29](https://user-images.githubusercontent.com/2589111/75352258-c0779080-58a9-11ea-96d4-cfea76434961.png)

or, copied pasted from PDF viewer

```
A search on the TEX Stack Exchange website for either ”expl3” or ”latex3” at time of writing yield around one thousand results each.
```

From [another thread](https://github.com/latex3/hyperref/issues/113) I believe there might be some "mark-up error" in the source. Perhaps the mark-up  should use `\enquote`? 

TBH, I have not looked further than this possibly isolated example.

**Comment by PhelypeOleinik** [comment]
Thanks for the report!

I moved this issue here, as the `l3news` are in this repository.

Yes, the sources use `"..."` instead of `\enquote`, in several places.

Though these documents are already published, so I'm not sure to
what extend is it valid to modify them... Oddly enough, the version
[published in TUGBoat](https://www.tug.org/TUGboat/tb35-1/tb109l3news.pdf) is correct:

![blub](https://user-images.githubusercontent.com/12678598/75353863-d590f700-588a-11ea-9276-e897fb5285d8.png)

**Comment by PhelypeOleinik** [comment]
> Yes, the sources use "..." instead of \enquote, in several places.

But it also contains:
```latex
\usepackage{csquotes}
\MakeOuterQuote{"}
```
so `"expl3"` prints as `“expl3”` (copied from the PDF viewer)...

Where did you get that screenshot from?

**Comment by jfbu** [comment]
> Where did you get that screenshot from?

I did `texdoc l3news` on a recently updated TL'2019. It is a PDF containing all issues (this may affect "Though these documents are already published,") Apple Preview reports a creation date of `21 févr. 2020 19:04`, with `pdfTeX-1.40.20` being the producer.

```
$ texdoc -l l3news
 1 /usr/local/texlive/2019/texmf-dist/doc/latex/l3kernel/l3news.pdf
Enter number of file to view, RET to view 1, anything else to skip: 
```

The file has 379209 bytes according to Adobe Reader who alsos says the producer is  `pdfTeX-1.40.20`. Was the document produced with OT1 encoding for TeX fonts?

**Comment by jfbu** [comment]
> from ltnews (issue 9, March 2014):

sorry for my typo, I meant `l3news` I mistyped. And I should have been clear I was referring to the whole pdf of all l3news issues. And I should not have posted on latex2e tracker, this was another mistake by inattention.

**Comment by PhelypeOleinik** [comment]
> I did `texdoc l3news`

Ah, I forgot about this one, sorry.

The “full” `l3news` is build with a master `l3news.tex` which inputs all
others, and disables package-loading, and, guess what, it didn't have
`\MakeOuterQuote` :-)

I'll fix that. Thanks again for the report.

> sorry for my typo, I meant l3news I mistyped. And I should have been clear I was referring to the whole pdf of all l3news issues. And I should not have posted on latex2e tracker, this was another mistake by inattention.

Don't worry about that :-)

**Comment by PhelypeOleinik** [comment]
Fixed with commit bbbda424
